akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -370,6 +370,12 @@ final class CoordinatedShutdown private[akka] (
   private val _jvmHooksLatch = new AtomicReference[CountDownLatch](new CountDownLatch(0))
   @volatile private var actorSystemJvmHook: OptionVal[Cancellable] = OptionVal.None
 
+  /**
+   * INTERNAL API
+   */
+  private[akka] lazy val terminationWatcher =
+    system.systemActorOf(CoordinatedShutdownTerminationWatcher.props, "coordinated-shutdown-termination-watcher")
```

Comment:
we probably have a mix of naming conventions for actor names, but camel case with lower first is what we typically prefer